From 616852cf2b1f235a5671013fbf3f1564be79ea1b Mon Sep 17 00:00:00 2001
From: RyuzakiKK <aasonykk@gmail.com>
Date: Fri, 7 Dec 2018 10:51:39 +0100
Subject: [PATCH 25/25] thunderbolt: fix missing variable underscore for
 `enrolling`

The variable `this.enrolling` is a typo because it has not been defined
before and is also never used.
`this._enrolling` is what it was meant to be.
---
 js/ui/status/thunderbolt.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/status/thunderbolt.js b/js/ui/status/thunderbolt.js
index 4584748ed..3d7a9614a 100644
--- a/js/ui/status/thunderbolt.js
+++ b/js/ui/status/thunderbolt.js
@@ -195,7 +195,7 @@ var AuthRobot = new Lang.Class({
 	if (this._enrolling)
 	    return;
 
-	this.enrolling = true;
+	this._enrolling = true;
 	GLib.idle_add(GLib.PRIORITY_DEFAULT,
 		      this._enrollDevicesIdle.bind(this));
     },
-- 
2.20.0

